docs/source/debugging.rst

```diff
@@ -0,0 +1,80 @@
+Debugging
+=========
+
+Run with the ``--log-level verbose`` CLI option to see more information in the output,
```

Comment:
--log-level verbose is now the default log level, may be worth mentioning that here.